packages/lodestar-cli/src/cmds/validator/paths.ts

```diff
@@ -0,0 +1,21 @@
+import path from "path";
+import {IGlobalArgs} from "../../options";
+
+export interface IValidatorPaths extends Pick<IGlobalArgs, "rootDir"> {
+  dbDir?: string;
+  validatorDir?: string;
+  keystoresDir?: string;
+  secretsDir?: string;
+}
+
+/**
+ * Defines the dynamic path structure of the validator files
+ */
+export function processValidatorPaths(options: IValidatorPaths): Required<IValidatorPaths> {
+  const rootDir = options.rootDir;
+  const dbDir = path.join(rootDir, options.dbDir || "validator-db");
```

Comment:
Can we put this inside the `validatorDir` on the same level as the keystores and secrets?